src/main/java/third_racing/Inputview.java

```diff
@@ -0,0 +1,48 @@
+package third_racing;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class Inputview {
```

Comment:
```suggestion
public class InputView {
```
클래스 이름은 카멜케이스 컨벤션을 지켜주세요!